addons/mail/models/discuss/ir_websocket.py

```diff
@@ -79,28 +68,72 @@ def _build_bus_channel_list(self, channels):
         domain = ["|", ("is_member", "=", True), ("id", "in", discuss_channel_ids)]
         all_user_channels = self.env["discuss.channel"].search(domain)
         member_specific_channels = [(c, "members") for c in all_user_channels if c.id not in discuss_channel_ids]
-        channels.extend([*all_user_channels, *member_specific_channels])
+        channels.extend([*all_user_channels, *member_specific_channels, *presence_channels])
         return super()._build_bus_channel_list(channels)
 
+    def _prepare_subscribe_data(self, channels, last):
```

Comment:
I feel like this method is not needed anymore. No override, just executing two one liners. What do you think?

Reply:
Sounds good, the override of `_build_bus_channel_list` takes care of building module-specific channels.
It also avoid repeated work